src/ingest_validation_tools/table-schemas/antibodies.yaml

```diff
@@ -0,0 +1,33 @@
+fields:
+  # NOTE: Example had "dataset_uuid": Won't work. Not known at submission time.
+  -
+    name: antibody_name
+    description: TODO
+  -
+    name: rrr_id
+    description: TODO
+    # TODO: pattern constraint
+  -
+    name: uniprot_accession_number
+    description: TODO
+    # TODO: pattern constraint
+  -
+    name: lot_number
+    description: TODO
+  -
+    name: validation_report_location
+    description: TODO
+    # Is this part of the submission? Or a URL? If a URL, put that in the field name.
```

Comment:
This is what we decided on while waiting for Bill Shirey to confirm:  users should create a directory called "extras" and put this Ab tsv and the Ab validation report in it. Would the proper field entry then be "/Ab_validation_report_123xyz.pdf" ?

Reply:
Ok, let's call this `validation_report_path` then, and if its in `extras`, the value would be `extras/ab-validation-report.pdf` or something like that. (Let's not use mixed case in filenames unless there's a reason.)